Router/ticketmaster.js

```diff
@@ -0,0 +1,157 @@
+import express from 'express'
+import listEndpoints from 'express-list-endpoints'
+import mongoose from 'mongoose'
+
+import Events from '../data/ticketmaster.json'
+
+const router = express.Router();
+
```

Comment:
When you already dissect it a little bit, maybe you will consider playing Abby Sciuto a little more and sending your Schemas to the separate classes in the `models` folder?